src/components/Accordion/Accordion.tsx

```diff
@@ -16,15 +17,31 @@ const generateSections = (sections, setExpanded, expanded, titleElement) =>
     />
   ));
 
+type Section = {
+  content: ReactNode;
+  key?: string;
+  onTitleClick?: MouseEvent;
+  title?: string;
+};
+
+type Props = {
+  className: string;
+  expanded: string;
+  externallyControlled: boolean;
+  sections: Section[];
+  onExpandedChange: (id, title: string) => void;
+  titleElement: HTMLHeadElement;
```

Comment:
I think `HTMLHeadElement` is the `<head>` tag, if you import the AccordionSection props it could end up being `titleElement: AccordionSection["titleElement"]`